desc/compute/_stability.py

```diff
@@ -232,3 +232,388 @@ def _magnetic_well(params, transforms, profiles, data, **kwargs):
         0,  # coefficient of limit is V_r / V_rr, rest is finite
     )
     return data
+
+
+@register_compute_fun(
+    name="ideal ball gamma2",
+    label="\\gamma_{\\mathrm{ballooning}}=-\\omega^2",
+    units="~",
+    units_long="None",
+    description="Normalized squared ideal ballooning growth rate, "
+    "requires data along a field line",
+    dim=1,
+    params=["Psi"],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="rtz",
+    data=[
+        "a",
+        "g^aa",
+        "g^ra",
+        "g^rr",
+        "cvdrift",
+        "cvdrift0",
+        "|B|",
+        "B^zeta",
+        "p_r",
+        "iota",
+        "shear",
+        "psi",
+        "psi_r",
+        "rho",
+    ],
+    source_grid_requirement={"coordinates": "raz", "is_meshgrid": True},
+    zeta0="array: points of vanishing integrated local shear to scan over. "
+    "Default 15 points linearly spaced in [-π/2,π/2]",
+)
+def _ideal_ballooning_gamma2(params, transforms, profiles, data, **kwargs):
+    """
+    Ideal-ballooning growth rate finder.
+
+    This function uses a finite-difference method
+    to calculate the maximum growth rate against the
+    infinite-n ideal ballooning mode. The equation being solved is
+
+    d/dζ(g dX/dζ) + c X = γ f X, g, f > 0
+
+    where
+
+    𝛋 = 𝐛 ⋅∇ 𝐛
+    g = a_N^3 * B_N * (b ⋅∇ζ) * |∇α|², / B,
```

Comment:
Is this the same g in the code?

Reply:
There are other terms that I missed in the docstring. I am adding them now.